### PR TITLE
feat(ui): Add a project list page for early adopters

### PR DESCRIPTION
**Pull request by macqueen**
i think i still maybe need to disable the button when people shouldnt' have project settings access, but lmk if there's other feedback in the meantime

screenshot 2018-03-16 17 44 31

### CODE REVIEW

**Comment by dcramer** [comment]
I think this is great (and more than enough to adapt to current use cases plus transition).

For the settings link we can just hide it if the user is missing 'project:write' scopes -- I think there's some easy examples of that in the previous iterations.

File: src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  renderNoProjects(teamSlug) {
+    return (
+      <tbody>
+        <tr>
+          <td>
+            <p className="project-list-empty">
+              {tct(
+                'There are no projects in this team. Get started by [link:creating your first project].',
+                {
+                  link: <a href={this.urlPrefix() + '/projects/new/?team=' + teamSlug} />,
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx]: id generally use the template syntax here (``` `${this.urlPrefix()}/projects/new/?team=${teamSlug}` ```)

File: src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          </Link>
+          <div className="project-chart">
+            {chartData && (
+              <LazyLoad>
+                <BarChart points={chartData} label="events" />
+              </LazyLoad>
+            )}
+          </div>
+        </Flex>
+        <Box ml={2}>
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx]: per your comment let's just kill this box when they dont have ``project:write`` permission

**Review by dcramer — CHANGES_REQUESTED** [review]
in general this looks great as a first pass -- once we get in the permission check i think we're ready to go

File: src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx (hunk 1 above)
Comment (macqueen) [reply src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx]: oh yeah i think i just blindly copy pasted from ExpandedTeamList, will fix

File: src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx (hunk 2 above)
Comment (macqueen) [reply src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx]: 👍 yeah i just wanted to get this up so you/chris could make sure it looks good. will add that now.

**Comment by macqueen** [comment]
@dcramer fixed those two things. one thing i just realized...i'm not accounting for teams that have no projects here since I was just getting the teams from the project list. do we care about showing teams without projects on this page? or should we just assume people can go to the org settings teams page to deal with that

**Comment by dcramer** [comment]
@macqueen we should account for them I think, but would you even end up in that situation? I guess the only way you'd access those projects is if you're a manager/owner, so we could consider them orphaned.

**Comment by macqueen** [comment]
@dcramer so i've accounted for projects without teams here just not teams without projects. but i feel like that's maybe ok since the main purpose for this page is to let people get to their projects/issues pages more easily?

**Comment by dcramer** [comment]
Oh yeah sorry that's 100% fine

**Review by dcramer — APPROVED** [review]

**Pull request merged**